docs/root/configuration/listener_filters/proxy_protocol.rst

```diff
@@ -0,0 +1,13 @@
+.. _config_listener_filters_proxy_protocol:
```

Comment:
Can we add a release note that we now support proxy protocol v2? Also, in this doc section should we make it more clear that we support both v1 and v2?

Reply:
echoing this, also calling out which part of the spec we implement (i.e. no TLV parsing)